src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/PosteriorLikelihoodsUtils.java

```diff
@@ -330,29 +294,31 @@ else if ( alleleCountsFromVCF instanceof String || alleleCountsFromVCF instanceo
      * @param integerListContainingVCField - Object returned by a call to VariantContext::getAttribute()
      * @return - array of ints
      */
+    @SuppressWarnings("unchecked")
     public static int[] extractInts(final Object integerListContainingVCField) {
```

Comment:
This is sort of gross, but I blame HTSJDK and its refusal to use the type information it already has from the annotation headers.  Maybe it's worth documenting that if we're working with a integerListContainingVCField that's read directly out of the file it will be a String but if it gets pulled from a VariantContext object built elsewhere in the code it will be an Integer or a List, depending on how many alleles there are at the site (if I recall correctly).